src/network/networkd-util.c

```diff
@@ -53,6 +53,30 @@ DEFINE_CONFIG_PARSE_ENUM(config_parse_link_local_address_family, link_local_addr
                          AddressFamily, "Failed to parse option");
 DEFINE_STRING_TABLE_LOOKUP(dhcp_lease_server_type, sd_dhcp_lease_server_type);
 
+int config_parse_address_family (
```

Comment:
Please use `DEFINE_CONFIG_PARSE_ENUM()` macro.

Reply:
I will get rid of this completely, looks like folks don't want auto-masquerade for ipv6 after all.